apps/test/util/testUtils.js

```diff
@@ -291,3 +291,30 @@ export function throwOnConsoleWarnings() {
     firstError = null;
   });
 }
+
+/**
+ * Stub everything on window.Applab for every test in this describe block.
+ */
+export function stubWindowApplab() {
```

Comment:
Better?  Simple `replaceOnWindow(key, temporaryObject)` and `restoreOnWindow(key)` helpers.